app/services/api_cost_tracker.py

```
@@ -0,0 +1,879 @@
+            
+            # Get current daily and monthly costs
+            daily_cost = float(await redis.get(f"cost:daily:{date_key}") or 0)
+            monthly_cost = 0.0
+            
+            # Calculate monthly cost by summing daily costs
+            for day in range(1, 32):  # Check up to 31 days
+                try:
+                    day_key = datetime.utcnow().replace(day=day).strftime("%Y-%m-%d")
+                    day_cost = float(await redis.get(f"cost:daily:{day_key}") or 0)
+                    monthly_cost += day_cost
+                except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)